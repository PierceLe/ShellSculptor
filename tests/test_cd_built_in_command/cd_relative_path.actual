>> cd ~/test_data/PWD_TEST/Hello
cd: no such file or directory: /Users/piercele/test_data/PWD_TEST/Hello
>> cd How/Are
cd: no such file or directory: How/Are
>> pwd
/Users/piercele/Desktop/year-2/sem1/info1112/ass1/ShellSculptor/tests

