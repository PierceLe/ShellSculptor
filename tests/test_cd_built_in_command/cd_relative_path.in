cd ~/test_data/PWD_TEST/Hello
cd How/Are
pwd
