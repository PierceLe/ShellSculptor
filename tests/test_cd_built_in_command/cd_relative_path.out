>> cd ~/test_data/PWD_TEST/Hello
>> cd How/Are
>> pwd
/home/test_data/PWD_TEST/Hello/How/Are

